core/textile/client.go

```diff
@@ -125,7 +129,28 @@ func (tc *textileClient) start(ctx context.Context, cfg config.Config) error {
 	tc.ConnectToHub(ctx)
 
 	tc.Ready <- true
-	return nil
+
+	// Healthcheck flow
+	tc.healthcheck(ctx)
+	for {
+		timeAfterNextCheck := 60 * time.Second
+		if tc.isConnectedToHub == false || tc.isInitialized == false {
+			timeAfterNextCheck = 5 * time.Second
```

Comment:
@dmerrill6 

Do you think there's a risk of flooding the hub with so many so frequent connections?

Reply:
I tested this a bunch and it's not flooding because the healthcheck stops after failing to get a public key, thus not even getting to do anything at the network. If there are keys and it passes this barrier, then it goes to `getHubCtx` which returns a cached data from the store so there shouldn't be flooding either. In any case I think we can test this and revisit if we find any issues.

Reply:
@dmerrill6 

Makes sense, thank you!